rclcpp_components/cmake/rclcpp_components_register_node.cmake

```diff
@@ -30,17 +30,17 @@ macro(rclcpp_components_register_node target)
   set(component ${ARGS_PLUGIN})
   set(node ${ARGS_EXECUTABLE})
   _rclcpp_components_register_package_hook()
-  set(_path "lib")  
+  set(_path "lib")
   set(library_name "$<TARGET_FILE_NAME:${target}>")
   if(WIN32)
     set(_path "bin")
   endif()
   set(_RCLCPP_COMPONENTS__NODES
     "${_RCLCPP_COMPONENTS__NODES}${component};${_path}/$<TARGET_FILE_NAME:${target}>\n")
   configure_file(${rclcpp_components_NODE_TEMPLATE}
-    ${PROJECT_BINARY_DIR}/node_main_configured.cpp.in)
+    ${PROJECT_BINARY_DIR}/rclcpp_components/node_main_configured_${node}.cpp.in)
```

Comment:
Looks like this change (and on L43) are already on master. Maybe this branch could use a rebase?

Reply:
@jacobperron  thinks its good onw